zebra-chain/src/parameters/network_upgrade.rs

```diff
@@ -216,6 +226,8 @@ pub(crate) const CONSENSUS_BRANCH_IDS: &[(NetworkUpgrade, ConsensusBranchId)] =
     (Canopy, ConsensusBranchId(0xe9ff75a6)),
     (Nu5, ConsensusBranchId(0xc2d6d0b4)),
     (Nu6, ConsensusBranchId(0xc8e71055)),
+    // FIXME: use a proper value below
+    (Nu7, ConsensusBranchId(0xc8e71056)),
```

Comment:
I think Zebra will panic if this doesn't match the one in `zcash_protocol`, so it would be good to generate a random one now, get started on the NU7 deployment ZIP, and make the change there as well (behind the compilation cfg flag).

Reply:
Opened a [PR adding a draft NU7 deployment ZIP](https://github.com/zcash/zips/pull/928) with `0x77190AD8` as the consensus branch id, it could still change, but we may as well use it here as it's much less likely to need an update later than a random number.